centrifuge-app/src/pages/Loan/ExternalRepayForm.tsx

```diff
@@ -187,94 +198,101 @@ export function ExternalRepayForm({ loan, destination }: { loan: ExternalLoan; d
   return (
     <FormikProvider value={repayForm}>
       <Stack as={Form} gap={3} noValidate ref={repayFormRef}>
-        <Stack gap={1}>
-          <Shelf gap={1}>
+        <Box
+          padding="24px 16px"
```

Comment:
```suggestion
          px={3}
          py={2}
```
prefer theme values over hardcoded ones